enterprise/server/cmd/ci_runner/main.go

```diff
@@ -129,33 +147,22 @@ func main() {
 	if err := ensurePath(); err != nil {
 		fatal(status.WrapError(err, "ensure PATH"))
 	}
-	if err := setupGitRepo(ctx); err != nil {
+	if err := ws.setup(ctx); err != nil {
 		fatal(status.WrapError(err, "failed to set up git repo"))
 	}
-	if err := os.Chdir(repoDirName); err != nil {
-		fatal(status.WrapErrorf(err, "cd %q", repoDirName))
-	}
-	runDebugCommand(&initLog, `pwd`)
-	runDebugCommand(&initLog, `ls -la`)
+	runDebugCommand(ws.log, `pwd`)
+	runDebugCommand(ws.log, `ls -la`)
```

Comment:
can we remove the runDebugCommand stuff now?

Reply:
Done